src/api/routes/tx.ts

```diff
@@ -90,6 +91,31 @@ export function createTxRouter(db: DataStore): RouterWithAsync {
     res.json(response);
   });
 
+  router.getAsync('/multiple', async (req, res, next) => {
+    const txList: string[] = req.query.tx_id as string[];
+    const eventLimit = parseTxQueryEventsLimit(req.query['event_limit'] ?? 96);
```

Comment:
Where does the 96 come from? Seems sensible to leave a short comment or something to know why this default exists.

Reply:
Basically, I replicated the functionality of `/extended/v1/tx/{tx_id}`:
https://github.com/blockstack/stacks-blockchain-api/blob/master/docs/openapi.yaml#L216

96 was specified here for this functionality: 
https://github.com/blockstack/stacks-blockchain-api/blob/master/src/api/routes/tx.ts#L216